datadog_checks_base/datadog_checks/base/checks/win/winpdh_base.py

```diff
@@ -154,24 +175,24 @@ def check(self, instance):
             for counter, values in list(iteritems(self._missing_counters)):
                 self._make_counters(counter_data=([counter], values))
 
-        for inst_name, dd_name, metric_func, counter in self._metrics[self.instance_hash]:
+        for inst_name, dd_name, metric_func, counterobj in self._metrics[self.instance_hash]:
```

Comment:
Had to use a different name than `counter` because that variable is used above to refer to a counter name (`str`, different type), and mypy doesn't like that (which is fair).